packages.json

```diff
@@ -4955,6 +4957,7 @@
   "https://github.com/way-to-code/git-macos.git",
   "https://github.com/wayfair-archive/prelude.git",
   "https://github.com/wearebeatcode/SwiftJailbreakDetection.git",
+  "https://github.com/web3swift-team/web3swift.git",
```

Comment:
This package is tripping up this PR at the moment. I've submitted a PR upstream: https://github.com/web3swift-team/web3swift/pull/665

I'll pull the other out into a new PR.